src/extensions/exception-autocapture/stack-trace.ts

```diff
@@ -51,7 +51,6 @@ export interface StackFrame {
 }
 
 const WEBPACK_ERROR_REGEXP = /\(error: (.*)\)/
-const STRIP_FRAME_REGEXP = /captureException/
```

Comment:
This is a superior approach but I couldn't get it to work. The function name I was seeing was `t.value` in the stack trace. Perhaps there's some Javascript function construction magic needed to ensure the function name gets caught in the stack. Open to any thoughts if people have them?

Reply:
probably because array.js is minified? Or the loaded extensions are if we're using the npm version of main package..

Reply:
Probably true - very meta given our current work 😂